services/movie.py

```diff
@@ -28,7 +32,8 @@
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
-    movie = Movie.objects.create(
+    with translation.atomic():
```

Comment:
The use of `translation.atomic()` is incorrect. It should be `transaction.atomic()` to ensure that the movie creation and association with genres and actors are done atomically.